scaler/handlers.go

```diff
@@ -38,18 +38,39 @@ func (e *impl) IsActive(
 	ctx context.Context,
 	scaledObject *externalscaler.ScaledObjectRef,
 ) (*externalscaler.IsActiveResponse, error) {
+	lggr := e.lggr.WithName("IsActive")
+	host, ok := scaledObject.ScalerMetadata["host"]
+	if !ok {
+		err := fmt.Errorf("no 'host' field found in ScaledObject metadata")
+		lggr.Error(err, "ScaledObjectRef", scaledObject)
+		return nil, err
+	}
+	if host == "interceptor" {
+		return &externalscaler.IsActiveResponse{
+			Result: true,
+		}, nil
+	}
+	allCounts := e.pinger.counts()
+	hostCount, ok := allCounts[host]
+	if !ok {
+		err := fmt.Errorf("host '%s' not found in counts", host)
+		lggr.Error(err, "allCounts", allCounts)
```

Comment:
can you log something more descriptive here?

```suggestion
		lggr.Error(err, "given host was not found", "host", host, "allCounts", allCounts)
```

Reply:
Maybe to be more descriptive, how about :
`lggr.Error(err, "Given host was not found in queue count map", "host", host, "allCounts", allCounts)`